README.md

```diff
@@ -43,3 +43,11 @@ devices](https://github.com/rust-vmm/vm-virtio/tree/main/crates/devices).
 This way a monolithic rust-vmm VMM implementation can reuse the core
 logic to service the virtio requests directly in the application.
 
+## Build dependency
```

Comment:
I guess this is because the distros don't package this yet as it is too new? Assuming one does not spam their system with sudo make install how can you point the build at a locally installed library?

Reply:
$ RUSTFLAGS='-L /home/<username>/libgpiod/lib/.libs/'  cargo build --release

Updated README with this now.

Reply:
I am not sure why distros don't keep it, but what we are using is surely very new and won't be available in distros.